packages/salesforcedx-vscode-soql/src/queryDataView/queryDataViewService.ts

```diff
@@ -129,7 +129,7 @@ export class QueryDataViewService {
       default:
         showAndTrackError(
           'data_view_message_type',
-          `Dataview unable to handle message type: ${type}`
+          `Error: ${type}. This is an unknown error.  Open an issue: https://github.com/forcedotcom/soql-tooling/issues/new/choose.`
```

Comment:
Emily - Updated error message

Reply:
Error: ${type}. Unknown error. Open an issue and supply the error message details: <link>.